ipset.go

```diff
@@ -114,6 +114,15 @@ func (s *IPSet) GetIPs(limit int) (ips []net.IP) {
 	return
 }
 
+// GetNetworks Retrieve a list of all networks included in the ipTree
```

Comment:
nit: Isn't the sentence supposed to begin with `GetNetworks`. IOW, should it be "GetNetworks retreives a list ..." At least that would be consistent with the methods around it.

Reply:
Got it thanks.